joomla/joomla-cms
# Redirect batch

The redirect component has a batch function to allow you to mass insert a set of urls. However this uses the common batch.php layout which has some javascript to ensure that you have selected one or more items to batch edit first. 

As the batch functionality in com_redirect works completely differently to all other batch buttons and does not require (or need) you to select an existing item first it was not working correctly.

To resolve this I created a new layout "batchredirect" which doesnt have the javascript check.
### Test instructions

Go to com_redirect and click the batch button on the toolbar - you will get a message about selecting an item first
Select an item and click the batch button. The modal will open and you can type new links on each line and when you click process you will see that new links have been created without any reference to the item you selected.

Apply the patch and you can click the batch button as intended without preselecting an irrelevant link first
Also check that this works when you do not have any redirects created.
